scss/_base_placeholders.scss

```diff
@@ -131,7 +139,7 @@
   }
 
   %u-no-margin--bottom {
-    &:not(h1):not(h2):not(h3):not(h4):not(h5):not(h6):not(p):not(small):not([class*='p-heading']) {
+    &:not(hr):not(h1):not(h2):not(h3):not(h4):not(h5):not(h6):not(p):not(small):not([class*='p-heading']) {
```

Comment:
I know it was not added in this PR, but wow, didn't know we have something like that...
This feels really arbitrary list. Hard to know if it's up to date. Would be better to have all no-margin--bottom related code in one place, not scattered exceptions around codebase.

Anyway, something for other time maybe.

Reply:
just wanted to avoid one !important overriding another !important - I gues it is not technically needed, the cascade can take care of that